tutorials/R/R-skills/Colorado-floods-data-visualization/COOP-NEIS-Precipitation-In-R/COOP-NEIS-Precipitation-In-R.md

```diff
@@ -568,12 +568,13 @@ Let's turn our plot into an interactive Plotly plot.
     # view plotly plot in R
     ggplotly(precPlot_flood2)
 
-    ## Warning in normalizePath(path.expand(path), winslash,
-    ## mustWork): path[1]="webshot1d202bb358c8.png": The system
+    ## Warning in normalizePath(path.expand(path),
+    ## winslash, mustWork):
+    ## path[1]="webshot47e06fcd5c8a.png": The system
     ## cannot find the file specified
 
     ## Warning in file(con, "rb"): cannot open file 'C:
-    ## \Users\fsanchez\AppData\Local\Temp\RtmpkhPoVW\file1d205b0049c0\webshot1d202bb358c8.png':
+    ## \Users\fsanchez\AppData\Local\Temp\Rtmp8mlAql\file47e025414f05\webshot47e06fcd5c8a.png':
     ## No such file or directory
```

Comment:
Seeing errors in this one too. Plot.ly can be really finicky within .Rmd. If I remember correctly, there was another tutorials that used plot.ly and I just made those chunks eval=FALSE, comment=NA so that the user could run those lines, but it did not embed the plot.ly plot within the .md or the tutorial page on the website. What do you think will be the best way to work around this?